src/NJsonSchema.CodeGeneration.CSharp/Models/EnumTemplateModel.cs

```diff
@@ -61,11 +72,16 @@ public IEnumerable<EnumerationItemModel> Enums
                             _schema.EnumerationNames.ElementAt(i) :
                             _schema.Type.HasFlag(JsonObjectType.Integer) ? "_" + value : value.ToString();
 
+                        string MyDesc = value.ToString();
```

Comment:
Use var, rename to "description"

Reply:
Only set description when it's actually available, no value fallback, then only render attribute if description is available (see below)